include/wil/resource.h

```diff
@@ -4186,6 +4186,7 @@ namespace wil
 #if !defined(NOUSER) && !defined(NOWH)
     typedef unique_any<HHOOK, decltype(&::UnhookWindowsHookEx), ::UnhookWindowsHookEx> unique_hhook;
 #endif
+    typedef unique_any<HWINEVENTHOOK, decltype(&::UnhookWinEvent), ::UnhookWinEvent> unique_hwineventhook;

```

Comment:
These are under `#ifndef NOWINABLE` in winuser.h.  Please add appropriate guards like the `#if !defined(NOUSER) && !defined(NOWH)` here.

Reply:
Thanks, fixed.